test/src/main/configurations/MainConfig/ConfigurationReplacerPipe.xml

```diff
@@ -7,7 +7,7 @@
 			<exits>
 				<exit path="EXIT" state="success" />
 			</exits>
-			<pipe name="Replace" className="org.frankframework.pipes.ReplacerPipe" find="hallo" replace="bonjour" replaceNonXmlChars="true" replaceNonXmlChar="x">
+			<pipe name="Replace" className="org.frankframework.pipes.ReplacerPipe" find="hallo" replace="bonjour" replaceNonXmlChars="true" nonXmlReplacementCharacter="x">
```

Comment:
This changes existing configurations without backwards-compatibility.

The new name is more logical, but for the configuration at least the old name should also be supported for backwards compatibility (the attribute can be marked as `@Deprecated` of course to steer users towards using the new name instead).